plugins/GrapesJsBuilderBundle/Assets/library/js/builder.js

```diff
@@ -37,7 +32,7 @@ function launchBuilderGrapesjs(formName) {
   const builder = new BuilderService(assetsConfig);
   // Initialize GrapesJS
   builder.initGrapesJS(formName);
-
+  Mautic.builder = builder;
```

Comment:
So far I could not find any better option than this. So I guess we can go with it for now.